src/main/java/org/openstreetmap/atlas/checks/validation/intersections/SelfIntersectingPolylineCheck.java

```diff
@@ -30,13 +32,13 @@
  *
  * @author mgostintsev
  * @author dbaah
+ * @author sayas01
  */
 public class SelfIntersectingPolylineCheck extends BaseCheck<Long>
 {
     private static final String AREA_INSTRUCTION = "Feature {0,number,#} has invalid geometry at {1}";
     private static final String POLYLINE_BUILDING_INSTRUCTION = "Feature {0,number,#} is a incomplete "
             + "building at {1}";
-
```

Comment:
Any reason for this change?

Reply:
I removed it as it was an extra line between the constants. I can revert it back as this is not part of the enhancement.